daemon/cmd/daemon_main.go

```diff
@@ -345,6 +345,9 @@ func InitGlobalFlags(cmd *cobra.Command, vp *viper.Viper) {
 	flags.Bool(option.EnableSocketLB, false, "Enable socket-based LB for E/W traffic")
 	option.BindEnv(vp, option.EnableSocketLB)
 
+	flags.Bool(option.EnableSocketLBPodConnectionTermination, true, "Enable terminating connections to deleted service backends when socket-LB is enabled")
```

Comment:
Question: Are there any use cases where users would want to disable this flag? Is this for the backward compatibility?

Reply:
I'd think we always want to have this enabled whenever users have KPR enabled. The flag to disable could also be hidden imho and does not necessarily need to be explicitly exposed via helm.

Reply:
>I'd think we always want to have this enabled whenever users have KPR enabled.

+1

I've also added the flag to Helm settings, but it's not documented so it's hidden.